test/features/caseList.feature

```diff
@@ -5,6 +5,10 @@ Feature: Set of Scenarios testing the functionality of the Case List table
     Given I have logged in
 
   Scenario: A Case Reference column in case list results has hyphens every 4 digits
+    Given the definition sheet 'CaseEventToFields' looks like this
```

Comment:
I think we should not mention or tie ourselves in any way to spreadsheets in our acceptance tests. We should be agnostic to the mechanism we use to define our definition. 
At the moment is in a spreadsheet, tomorrow it could be json based, or via the Admin Web UI only. But anyway this is not important here. 
I think we spoke about this before. 
It's very easy to refactor this in a more abstract language. And I strongly suggest we do.
Something like:
`Given the definition for 'event fields' looks like this` 

When we implement these steps, we will never interact with the spreadsheet directly. We will retrieve the definition from the database to validate what we declared
here is what is on the definition database. How it ended up there (via spreadsheet upload at the moment) we don't care, and should not be mentioned here.